sqlalchemy_cockroachdb/requirements.py

```diff
@@ -161,6 +161,19 @@ def sync_driver(self):
             lambda config: config.db.dialect.is_async
         )
 
+    @property
+    def array_type(self):
+        # DDL like
+        #
+        # CREATE TABLE foo (thing INTEGER[])
```

Comment:
hm i'm surprised. i can run this command fine (using `cockroach sql` to connect): `CREATE TABLE foo (thing INTEGER[]);`